sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/WebPubSubHub.cs

```diff
@@ -18,15 +18,21 @@ public abstract class WebPubSubHub
         /// <param name="request"><see cref="ConnectedEventRequest"/> to get client connect request information.</param>
         /// <param name="cancellationToken">The <see cref="CancellationToken"/> used to cancel the request.</param>
         /// <returns>Build a valid <see cref="WebPubSubEventResponse"/> with ConnectEventRequest.CreateResponse().</returns>
-        public abstract ValueTask<WebPubSubEventResponse> OnConnectAsync(ConnectEventRequest request, CancellationToken cancellationToken);
+        public virtual ValueTask<WebPubSubEventResponse> OnConnectAsync(ConnectEventRequest request, CancellationToken cancellationToken)
+        {
+            return default;
```

Comment:
How about by default returning a `WebPubSubEventResponse.Success`

Reply:
`default` is body null and status code 200, same as `Success`. Do you suggest to specifically define the `Success` for users to easy understand?

Reply:
We have the `CreateResponse()` methods for user defined parameters in normal cases, and in null case, user can return `base.OnConnectAsync(request, ctk)`. Not much need to define `Success` anymore.